README.md

```diff
@@ -250,9 +250,9 @@ unmock. nock('http://myapp.iriscouch.com')
   })
 ```
 
-### Passthrough
+### Trivial API calls
 
-For "boring" API calls where you are just passing through information, you can instruct unmock to serve random `200` resposnes to those requests using `tldr`.
+For "trivial" API calls where you are just passing through information, you can instruct unmock to serve random `200` resposnes to those requests using `tldr`.
```

Comment:
A few notes about this section...

1) Can we remove `just`? I think the sentence works without it and this language is often considered condescending in documentation. We could also reword it to something like `For "trivial" API calls where the goal is to pass through information...` 🤔 

2) Typo --> `responses`